st2common/st2common/util/action_db.py

```diff
@@ -195,6 +196,11 @@ def update_liveaction_status(status=None, result=None, context=None, end_timesta
     old_status = liveaction_db.status
     liveaction_db.status = status
 
+    liveaction_db.log.append({
```

Comment:
To avoid race issues, we should use MongoDB method for automatically publishing an item to an array.